tests/postgres/describe.rs

```diff
@@ -84,7 +84,7 @@ async fn it_describes_composite() -> anyhow::Result<()> {
 
     let ty = d.columns()[0].type_info();
 
-    assert_eq!(ty.name(), "inventory_item");
+    assert_eq!(ty.name(), "public.inventory_item");
```

Comment:
There should be tests involving cases where the database will forcibly escape names, like `"my::schema"."TypeName"`.